lib/frosty_friend_web/controllers/charts.html.heex

```diff
@@ -1,5 +1,24 @@
 <body>
-  <div style="width: 800px;">
-    <canvas id="line_chart" phx-hook="ChartJS" data-chart-data={Jason.encode!(@chart_data)} data-element="line_chart"></canvas>
+<div class="flex flex-col gap-10 m-0 h-full">
+  <div class="flex grow w-auto">
+    <canvas
+      id="line_chart"
+      phx-hook="ChartJS"
+      data-chart-data={Jason.encode!(@chart_data)}
+      data-element="line_chart"
+      data-type="line"
+    >
+    </canvas>
   </div>
+  <div class="flex grow w-auto">
+    <canvas
+      id="bar_chart"
+      phx-hook="ChartJS"
+      data-chart-data={Jason.encode!(@chart_data)}
+      data-element="bar_chart"
+      data-type="bar"
+    >
+    </canvas>
+  </div>
```

Comment:
instead of two charts we should just have a toolbar with toggleable options

Reply:
Okay sounds good!